client/Makefile

```diff
@@ -117,8 +125,13 @@ INCLUDES += $(HARDNESTEDLIBINC)
 
 ## Lua
 ifneq ($(SKIPLUASYSTEM),1)
-    LUAINCLUDES = $(shell $(PKG_CONFIG_ENV) pkg-config --cflags lua5.2 2>/dev/null)
-    LUALDLIBS = $(shell $(PKG_CONFIG_ENV) pkg-config --libs lua5.2 2>/dev/null)
+    ifdef MACPORTS_PREFIX
+        LUAINCLUDES = $(shell $(PKG_CONFIG_ENV) pkg-config --cflags lua-5.2 2>/dev/null)
+        LUALDLIBS = $(shell $(PKG_CONFIG_ENV) pkg-config --libs lua-5.2 2>/dev/null)
+    else
+        LUAINCLUDES = $(shell $(PKG_CONFIG_ENV) pkg-config --cflags lua5.2 2>/dev/null)
+        LUALDLIBS = $(shell $(PKG_CONFIG_ENV) pkg-config --libs lua5.2 2>/dev/null)
+    endif
```

Comment:
What is the difference that we need to have this IFDEF ?   It looks the same on both sides.

Reply:
THere is a diff, I see a lua-5.2 and a lua5.2

Reply:
Sneaky dash.  Now I see it too.   Thanks!